<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QComboBox {
	background-color:rgba(0,0,0,0);
	border:2px solid rgba(0,0,0,0);
	border-bottom-color:rgba(46,82,101,200);
	color:rgb(0,0,0);
	padding-bottom:7px;
}

#btnCadastrarCurso {
	background-color: #01498a;
	color: #FFF;
border: none;
border-radius: 15px;
}

#btnCadastrarCurso:hover {
	background-color: #f6921e;
}

#respostas {
	color: red;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>CADASTRO DE CURSOS</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="nomeCurso">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Nome do Curso</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>213</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Período do Curso</string>
      </property>
     </widget>
     <widget class="QLabel" name="respostaCadastroIncompleto">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>500</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="respostaCadastrando">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>500</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="periodoCurso">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Manhã</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tarde</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Noite</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="ofertaCurso">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Oferta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cargaCurso">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Carga Horária</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quantidadeAlunos">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Quantidade de Alunos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="horasPorDia">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Horas por dia</string>
      </property>
     </widget>
     <widget class="QComboBox" name="campoArea">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>Área</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Área</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCadastrarCurso">
     <property name="minimumSize">
      <size>
       <width>762</width>
       <height>47</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="respostas">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nomeCurso</tabstop>
  <tabstop>ofertaCurso</tabstop>
  <tabstop>campoArea</tabstop>
  <tabstop>periodoCurso</tabstop>
  <tabstop>cargaCurso</tabstop>
  <tabstop>horasPorDia</tabstop>
  <tabstop>quantidadeAlunos</tabstop>
  <tabstop>btnCadastrarCurso</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
